tests/test_core.py

```diff
@@ -8469,7 +8469,7 @@ def setUp(self):
 # Main wasm test modes
 wasm0 = make_run('wasm0', emcc_args=['-O0'])
 wasm0g = make_run('wasm0g', emcc_args=['-O0', '-g'])
-wasm1 = make_run('wasm1', emcc_args=['-O1'])
+wasm1 = make_run('wasm1', emcc_args=['-O1', '-g'])
```

Comment:
```suggestion
wasm1 = make_run('wasm1', emcc_args=['-O1'])
wasm1g = make_run('wasm1', emcc_args=['-O1', '-g'])
```
(I assume the intent was to add a variant with g, not replace the existing one?)

Reply:
Reverted.